feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -307,7 +310,9 @@ class PreviewViewModel @Inject constructor(
     }
 
     sealed interface ImageCaptureEvent {
-        object ImageSaved : ImageCaptureEvent
+        data class ImageSaved(
+            val savedUri: Uri? = null
```

Comment:
Can this now be made non-null?

Reply:
This is null in the case of external capture right now